jobs/service-fabrik-broker/spec

```diff
@@ -7,6 +7,7 @@ packages:
   - service-fabrik-broker
   - jq
   - yaml2json
+  - ha-utils
```

Comment:
why broker need to have this? We won't keep any ha footprint in broker job.

Reply:
yes.. will be getting rid of this.